package.json

```diff
@@ -42,9 +44,11 @@
     "react-native-gesture-handler": "~1.10.2",
     "react-native-get-random-values": "~1.7.0",
     "react-native-paper": "^4.9.2",
+    "react-native-push-notification": "^8.0.0",
```

Comment:
Seems like you aren't using this anymore? And went with `expo-notifications`, do we still need this? If not please remove

Reply:
cleared